.github/workflows/track_todo.yml

```diff
@@ -0,0 +1,17 @@
+name: Track TODOs
+on:
+  push:
+    branches:
+      - develop
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Run tdg-github-action
+      uses: ribtoks/tdg-github-action@master
+      with:
```

Comment:
Let's add a `Label` with name TODO so we can easily track these kind of issues cc @mrinalwadhwa